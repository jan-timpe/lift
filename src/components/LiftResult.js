import React, { Component } from 'react'
import '../style/LiftResult.css'


export default class LiftResult extends Component {

  render() {
    return (
      <div className="LiftResult">
        <p>{this.props.result.weight} on {this.props.result.date}</p>
      </div>
    )
  }

}