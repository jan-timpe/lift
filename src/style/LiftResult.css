.LiftResult {
  padding: 8px;
}