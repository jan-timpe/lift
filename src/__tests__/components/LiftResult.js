import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LiftResult from '../../components/LiftResult'


describe(`LiftResult`, () => {

  // snapshot
  it(`should render`, () => {
    const result = {
      id: 0,
      liftId: 0,
      weight: 200,
      date: '7/1/2018'
    }
    const wrapper = shallow(<LiftResult result={result} />)
    const component = wrapper
    expect(toJson(component)).toMatchSnapshot()
  })

})