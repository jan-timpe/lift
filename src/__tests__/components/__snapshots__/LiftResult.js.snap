// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LiftResult should render 1`] = `
<div
  className="LiftResult"
>
  <p>
    200
     on 
    7/1/2018
  </p>
</div>
`;
